config/more_samples-glyf_colr_1.py

```diff
@@ -32,6 +34,11 @@ class SampleGlyph(NamedTuple):
     colr: Optional[Mapping[str, Any]] = None
 
 
+def _accessors():
+    for c in chain(range(ord("A"), ord("Z") + 1), range(ord("a"), ord("z") + 1)):
```

Comment:
you could `yield from string.ascii_letters` constants defined in the `string` built-in module: https://docs.python.org/3/library/string.html#string.ascii_letters

Reply:
Done, and this actually revealed above in how I used yield and the generator, should be fixed now.